widgets/BucketSinging.js

```diff
@@ -346,7 +349,7 @@ async function initContext(){
     }
 
     div.empty();
-    div.append("<p>That's enough singing.  Calibration is done.  On with the main event.</p>");
+    div.append("<p>That's enough singing.  Calibration is done.  If you experience difficulties during the event, you can refresh the page at any time and still keep your calibration settings.  On with the main event.</p>");
```

Comment:
This is good, I think, but it wasn't quite what I meant. What I meant was that in the "I guess no one will hear me" dialog, which a lot of people will see when they join if they didn't click the grant-permissions button fast enough, it would be good to say that refreshing *right now* and trying again would be a good idea. The reason it is a good idea is because the browser will remember that permission was granted, and so they will not have to win the race a second time.